Jenkinsfile

```diff
@@ -11,9 +11,9 @@ if (master_branches.contains(env.BRANCH_NAME)) {
     ])
 }
 
-task_wrapper('mesos-sec', master_branches){
+task_wrapper('mesos-sec', master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#dcos-security-ci') {
```

Comment:
Is there any way for me to verify that this is the right code?

Reply:
Why is it this channel? I had thought that this channel was for security related projects.

Reply:
> Is there any way for me to verify that this is the right code?
This has already been verified manually while merging the changes into master. This PR is just a backport.

> Why is it this channel? I had thought that this channel was for security related projects.
Currently, only the Admin Router is using Jenkinsfile-based CI. The DC/OS integration tests use Team City.

Reply:
Thanks!

Reply:
No need now, but FTR I think that a comment to that affect would be useful. Maybe next time you touch this code if you remember and agree 😄

Reply:
I can try to bundle this change together with the one @karlkfi mentioned (symbolic names for slack tokens). This will not happen soon as my main priority is shipping auto-unseling for dcos-secrets in 1.11.